app/assets/stylesheets/modules/gem.css

```diff
@@ -96,6 +96,14 @@
   font-size: 18px;
   color: #141c22; }
 
+.gem__rubygem-version-age {
+  margin-top: 10px;
+  margin-bottom: 30px;
+  display: block;
```

Comment:
Instead of this, maybe use a div instead of a span?

Reply:
it follows the rest, IMHO ok